packages/block-library/src/table/style.scss

```diff
@@ -76,6 +76,18 @@
 		th,
 		td {
 			border-color: transparent;
+
+			&.has-text-align-left {
```

Comment:
Do we really need these classNames to be defined here? I thought these were meant to be global classNames?

Reply:
Well spotted, I didn't realise there were global styles. Removed those styles.